aspnetcore/blazor/javascript-interop.md

```diff
@@ -143,10 +143,25 @@ The following example shows capturing a reference to the `username` `<input>` el
 }
 ```
 
-> [!NOTE]
-> Do **not** use captured element references as a way of populating the DOM. Doing so may interfere with the declarative rendering model.
-
-As far as .NET code is concerned, an `ElementReference` is an opaque handle. The *only* thing you can do with `ElementReference` is pass it through to JavaScript code via JavaScript interop. When you do so, the JavaScript-side code receives an `HTMLElement` instance, which it can use with normal DOM APIs.
+> [!WARNING]
+> Only use an element reference to mutate the contents of an empty element that doesn't interact with Blazor. This scenario is useful when a 3rd party API supplies content to the element. Because Blazor doesn't interact with the element, there's no possibility of a conflict between Blazor's representation of the element and the DOM.
+>
+> Do **not** use a captured element reference as a way of *populating* the DOM. Doing so may interfere with the declarative rendering model.
```

Comment:
This line seems to contradict the previous statement about mutating the contents of an empty element. I don't really see a different between populating the DOM and mutating the DOM. I think we can remove this sentence as it is overly conservative.